cukes-graphql/src/main/java/lv/ctco/cukes/graphql/internal/PreprocessGraphQLRequestBody.java

```diff
@@ -35,6 +40,12 @@ public void afterScenario() {
 
     @Override
     public void beforeRequest(RequestSpecification requestSpecification) {
```

Comment:
Please extract `requestFacade.getGraphQLRequest()` to a separate variable. It repeats 5 times in one method.

Reply:
done